challenge-03/README.md

```diff
@@ -150,8 +178,26 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+pessoa.apresentacao = function () {
+	if (pessoa.sexo === 'masculino') {
+		var artigo = 'o';
+	} else {
+		var artigo = 'a';
```

Comment:
-1
Declare as variáveis somente uma vez, no início da função com um valor padrão. Assim você consegue usar só o `if` para modificar o valor dela se necessário, se precisar do `else`, deixando o código bem mais limpo :)